core.php

```diff
@@ -65,12 +65,30 @@
 
 ob_start();
 
-# Load Composer autoloader
-require_once( dirname( __FILE__ ) . DIRECTORY_SEPARATOR . 'vendor/autoload.php' );
-
 # Load supplied constants
 require_once( dirname( __FILE__ ) . DIRECTORY_SEPARATOR . 'core' . DIRECTORY_SEPARATOR . 'constant_inc.php' );
```

Comment:
Including core.php typically includes a lot of files.  If we have a dependency on a new version of PHP, there can be a syntax error there.  It would be great if we can check php version before including core.php.

Reply:
> Including core.php typically includes a lot of files. If we have a dependency on a new version of PHP, there can be a syntax error there

That's exactly why I opened this PR. There was a syntax erorr in guzzle when trying to run Mantis on PHP 5.3

> It would be great if we can check php version before including core.php.

Before including? Wouldn't this mean that the checks had to be added to nearly all our php files?